File: CalculateMVVM/server/Calculator/src/main/java/com/lbc/calculator/domain/Problem.java
package com.lbc.calculator.domain;

public class Problem {
    public int problemNumber;
    public String problemContent;
    public String correctAnswer;


	public Problem(String problemContent, String correctAnswer) {
		super();
		this.problemContent = problemContent;
		this.correctAnswer = correctAnswer;
	}

	public int getProblemNumber() {
		return problemNumber;
	}

	public String getProblemContent() {
		return problemContent;
	}

	public String getCorrectAnswer() {
		return correctAnswer;
	}

	public Problem(int problemNumber) {
		super();
		this.problemNumber = problemNumber;
	}

	public Problem(String problemContent) {
		super();
		this.problemContent = problemContent;
	}
    
    
}

File: CalculateMVVM/app/CalculatorMVVM/app/src/main/java/com/lbc/practice/calculator/data/Problem.kt
package com.lbc.practice.calculator.data

class Problem {
    var problemNumber: Int?= null
    var problemContent: String? =null
    var correctAnswer: String? =null

    constructor(problemNumber: Int?) {
        this.problemNumber = problemNumber
    }

    constructor(problemContent: String?, correctAnswer: String?) {
        this.problemContent = problemContent
        this.correctAnswer = correctAnswer
    }

}